sql/telemetry_derived/error_aggregates/query.sql

```diff
@@ -0,0 +1,233 @@
+    0 AS usage_hours,
+    0 AS gpu_crashes,
+    0 AS plugin_crashes,
+    0 AS gmplugin_crashes
+  FROM
+    crash_pings
+),
+-- Get desktop usage hours and gpu, plugin, and gmplugin crashes from main pings
+main_ping_data AS (
+  SELECT
+    submission_timestamp,
+    normalized_channel AS channel,
+    environment.build.version,
+    application.display_version,
+    environment.build.build_id,
+    metadata.uri.app_name,
+    environment.system.os.name AS os,
+    environment.system.os.version AS os_version,
+    environment.build.architecture,
+    normalized_country_code AS country,
+    -- 0 for values retrieved from crash pings
+    0 AS main_crash,
+    0 AS content_crash,
+    0 AS startup_crash,
+    0 AS content_shutdown_crash,
+    LEAST(GREATEST(payload.info.subsession_length / 3600, 0), 25) AS usage_hours,  -- protect against extreme values
+    COALESCE(udf_keyed_histogram_get_sum(payload.keyed_histograms.subprocess_crashes_with_dump, 'gpu'), 0) AS gpu_crashes,
+    COALESCE(udf_keyed_histogram_get_sum(payload.keyed_histograms.subprocess_crashes_with_dump, 'plugin'), 0) AS plugin_crashes,
+    COALESCE(udf_keyed_histogram_get_sum(payload.keyed_histograms.subprocess_crashes_with_dump, 'gmplugin'), 0) AS gmplugin_crashes
+  FROM
+    main_pings
+),
+-- Get mobile usage hours from core pings
+core_ping_data AS (
+  SELECT
+    submission_timestamp,
+    normalized_channel AS channel,
+    metadata.uri.app_version AS version,
+    COALESCE(display_version, metadata.uri.app_version) AS display_version,
+    metadata.uri.app_build_id AS build_id,
+    metadata.uri.app_name,
+    os,
+    osversion AS os_version,
+    arch AS architecture,
+    normalized_country_code AS country,
+    -- 0 for values retrieved from crash pings
+    0 AS main_crash,
+    0 AS content_crash,
+    0 AS startup_crash,
+    0 AS content_shutdown_crash,
+    LEAST(GREATEST(durations / 3600, 0), 25) AS usage_hours,  -- protect against extreme values
+    0 AS gpu_crashes,
+    0 AS plugin_crashes,
+    0 AS gmplugin_crashes
+  FROM
+    core_pings
+),
+combined_ping_data AS (
+  SELECT
+    *
+  FROM
+    crash_ping_data
+  UNION ALL
+  SELECT
+    *
+  FROM
+    main_ping_data
+  UNION ALL
+  SELECT
+    *
+  FROM
+    core_ping_data
+)
+
+SELECT
+  DATE(submission_timestamp) AS submission_date,
+  udf_round_timestamp_to_minute(submission_timestamp, 5) AS window_start,
+  channel,
+  version,
+  display_version,
+  build_id,
+  app_name as application,
+  os as os_name,
+  os_version,
+  architecture,
+  country,
+  COUNT(*) AS count,
+  SUM(main_crash) AS main_crashes,
+  SUM(content_crash) AS content_crashes,
+  SUM(startup_crash) AS startup_crashes,
+  SUM(content_shutdown_crash) AS content_shutdown_crashes,
+  SUM(gpu_crashes) AS gpu_crashes,
+  SUM(plugin_crashes) AS plugin_crashes,
+  SUM(gmplugin_crashes) AS gmplugin_crashes,
+  SUM(usage_hours) AS usage_hours
+FROM
+  combined_ping_data
+WHERE
+  DATE(submission_timestamp) = @submission_date
```

Comment:
Is this going to be scheduled on Airflow?
If so, it would need to run more often than daily. If I remember correctly @wlach wanted to get updates every hour?

Reply:
Yes we'll run it every few hours in Airflow.  I don't expect any issues with running it more than once a day.